nekoyume/Assets/_Scripts/UI/Module/AcquisitionPlaceDescription.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using Nekoyume.EnumType;
+using Unity.Mathematics;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Nekoyume.UI.Module
+{
+    public class AcquisitionPlaceDescription : MonoBehaviour
+    {
+        [SerializeField]
+        private RectTransform leftDeco;
+
+        [SerializeField]
+        private RectTransform rightDeco;
+
+        [SerializeField]
+        private RectTransform contentsRectTransform;
+
+        [SerializeField]
+        private Button bgButton;
+
+        private static Camera _mainCamera;
+
+        private void Awake()
+        {
+            _mainCamera = Camera.main;
+            bgButton.onClick.AddListener(() => gameObject.SetActive(false));
+        }
+
+        public void Show(RectTransform parentTransform, RectTransform helpButtonTransform)
+        {
+            gameObject.SetActive(true);
+            var parentWorldPos = parentTransform.GetWorldPositionOfCenter();
+            var parentScreenPos = _mainCamera.WorldToScreenPoint(parentWorldPos);
+            if (parentScreenPos.x <= _mainCamera.pixelWidth / 2)
+            {
+                leftDeco.gameObject.SetActive(true);
+                rightDeco.gameObject.SetActive(false);
+                contentsRectTransform.MoveToRelatedPosition(helpButtonTransform, PivotPresetType.BottomRight, new float2(200f, 0f));
```

Comment:
Is it ok to move with constant parameter?

Reply:
OK. I'll reflect your sugestion. THX 👍

Reply:
I mean, i think it should handle various screen resolution.

Reply:
ah ok I understand... I'll look for another way. Thank you.

Reply:
@Unengine I tested it. The result was good. I confirmed that there is no problem with various resolutions such as FHD, QHD, and 4K.